docs/TRANSLATIONS.md

```diff
@@ -0,0 +1,179 @@
+
+- **Keep Placeholders Intact:** Do not translate or modify placeholders. Ensure they remain exactly as they are in the original key.
+* **For variables:** Strings like `:count`, `:total`, or `:gameTitle` should be kept as-is.
+* **For HTML-like tags:** Tags like `<0>`, `</0>`, etc, must remain intact and should be placed appropriately in the translated sentence.
+
+**Examples:**
+
+* For variables:
+```json
+{
+  "Next :count": "Próximo :count"
+}
+```
+* For tags:
+```json
+{
+  "Click <0>here</0> to visit the page.": "Clique <0>aqui</0> para visitar a página."
+}
+```
+
+- **Proper Placement:** Ensure placeholders are positioned correctly within the translated string to maintain the intended meaning. You can move placeholders as appropriate for your target language.
+
+## Submitting Your Changes
+
+1. **Locate the Correct JSON File:**
+
+Navigate to the _lang_ directory and open the JSON file corresponding to your target language. If the file does not exist, you can create a new one following the existing naming convention.
+
+**Example:**
+
+```
+lang/
+└── pt_BR.json
+```
+
+2. **Translate the Values:**
+
+- Open the JSON file in a text editor or an IDE.
+- Translate each value while keeping the keys unchanged.
+- Ensure that the JSON syntax remains valid (commas, quotes, brackets).
+
+3. **Validate JSON Syntax:**
+
+Use a JSON validator to ensure there are no syntax errors. Tools like [JSONLint](https://jsonlint.com/) can be helpful.
+
+4. **Commit Your Changes:**
+
+```shell
+git add .
+git commit -m "chore: update translations for pt_BR"
+```
+
+5. **Push to Your Fork:**
+
+```shell
+git push --set-upstream origin translate/pt_BR
+```
+
+6. **Open a Pull Request:**
+
+- Navigate to the [RAWeb repository](https://github.com/RetroAchievements/RAWeb) repository on GitHub.
+- Click the **Compare & pull request** button.
+- Provide a clear title and description for your pull request.
+- Submit the pull request for review.
+
+## Review Process
+
+Once your pull request is submitted:
+
+1. **Automated Checks:**
+
+The repository's CI/CD pipeline will run automated tests to ensure there are no syntax errors in your JSON files.
+
+2. **Manual Review:**
+
+A maintainer will review your translations for adherence to these guidelines.
+
+3. **Feedback:**
+
+You may receive feedback or requests for revisions. Address any feedback promptly to facilitate merging your changes.
+
+4. **Merging:**
+
+Once approved, your translations will be merged into the `master` branch and deployed with the next site release.
+
+## Deleting Your Branch
+
+Once your pull request is merged, you can safely delete the branch you created for the translation work.
+
+1. **Delete the Branch Locally:**
+
+```shell
+git branch -d translate/pt_BR
+```
+
+2. **Delete the Branch from Your Fork on GitHub:**
+
+```shell
+git push origin --delete translate/pt_BR
```

Comment:
It's much easier to just delete the branch from the PR. After the PR is merged, there's a Delete Branch button on the PR for the owner of the repository the PR was created from.

Reply:
Updated in latest to reflect this.